From 03cc85f4ac3d5b663d6221d31f6cbd738b9a1f01 Mon Sep 17 00:00:00 2001
From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Fri, 9 Feb 2018 23:48:32 +0100
Subject: [PATCH] solenv: Fix gb_MoTarget_get_install_target argument

This fixes l10n-*.files generation of packageinfo and makes
"install-packages-l10n-*" targets work again.

Change-Id: Ib4d726f1079360cade8d5f398b1f5144d6693152
---
 solenv/gbuild/extensions/post_PackageInfo.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/solenv/gbuild/extensions/post_PackageInfo.mk b/solenv/gbuild/extensions/post_PackageInfo.mk
index d0e0f15df133..a3bfa4ca7bb9 100644
--- a/solenv/gbuild/extensions/post_PackageInfo.mk
+++ b/solenv/gbuild/extensions/post_PackageInfo.mk
@@ -55,7 +55,7 @@ define gb_PackageInfo_emit_l10n_for_one_alllangpackage
 endef
 
 define gb_PackageInfo_emit_l10n_for_one_mo
-@echo "$(patsubst $(INSTDIR)/%,%,$(call gb_MoTarget_get_install_target,$(1)$(2)))" >> $(gb_PackageInfo_get_target)/l10n-$(2).files
+@echo "$(patsubst $(INSTDIR)/%,%,$(call gb_MoTarget_get_install_target,$(shell $(SRCDIR)/solenv/bin/localestr $(2))/LC_MESSAGES/$(1)))" >> $(gb_PackageInfo_get_target)/l10n-$(2).files
 
 endef
 
-- 
2.15.1

